release-notes/System.ServiceModel.Duplex.md

```diff
@@ -0,0 +1,15 @@
+# System.ServiceModel.Duplex
+
+## Information
+
+* [NuGet Package](https://nuget.org/packages/System.ServiceModel.Duplex)
+* [Report an issue](http://github.com/dotnet/wcf/issues/new)
+* [Source](https://github.com/dotnet/wcf/tree/master/src)
```

Comment:
Feels like the "Report an issue" and "Source" links are redundant. They are not component specific information. If someone can land on this page, it's more likely they can figure out how to report an issue or find our source. If not, our main page and documentation also have this information. 

I hope we can make these pages very concise, so users can find useful information with a glance (the less reading the better).